library/src/lib/localizator-editor/localization-editor-main/localization-editor-main.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, ContentChild, Input, OnInit, TemplateRef } from '@angular/core';
+import { LocalizationEditorInputDirective, LocalizationEditorLabel } from '../localization-editor.directives';
+
+/**
+ *  Component that represents main field with add-on and is always visible
+ *  ```html
+ *  <fd-localization-editor-main [label]="'EN'">
+ *       <input fd-localization-editor-input type="text" placeholder="EN">
+ *  </fd-localization-editor-main>
+ *  ```
+ */
+@Component({
+    selector: 'fd-localization-editor-main',
+    templateUrl: './localization-editor-main.component.html'
+})
+export class LocalizationEditorMainComponent implements OnInit {
+
+    /** The text for the right add-on. */
+    @Input() label: string;
+
+    /** @hidden */
+    type: string;
+
+    /** @hidden */
+    compact: boolean;
```

Comment:
can this be private?

Reply:
It's used in template.

Reply:
OK that's fine, I posted those "can this be private?" before looking at the template and if I see a public @hidden variable its always a flag for me lol